components/PageControl/examples/PageControlTypicalUseExample.swift

```diff
@@ -20,47 +20,51 @@ import MaterialComponents.MaterialPageControl
 
 class PageControlSwiftExampleViewController: UIViewController, UIScrollViewDelegate {
 
+  static let pageColors = [
+    MDCPalette.cyan.tint300,
+    MDCPalette.cyan.tint500,
+    MDCPalette.cyan.tint700,
+    MDCPalette.cyan.tint300,
+    MDCPalette.cyan.tint500,
+    MDCPalette.cyan.tint700
+  ]
+
   let pageControl = MDCPageControl()
   let scrollView = UIScrollView()
-  let pages = NSMutableArray()
+  let pages: [UIView] = PageControlSwiftExampleViewController.pageColors.enumerated().map {
+      enumeration in
+    let (i, pageColor) = enumeration
+    let page = UILabel()
+    page.text = "Page \(i + 1)"
+    page.font = page.font.withSize(50)
+    page.textColor = UIColor.init(white: 0, alpha: 0.8)
```

Comment:
is the `0.8` safe for compiling to a 32-bit machine? (Asking. I don't know answer.)

Reply:
Based on [this tool](https://www.h-schmidt.net/FloatConverter/IEEE754.html) it's stored as `0.800000011920928955078125`. [It currently is using that value](https://github.com/material-components/material-components-ios/blob/develop/components/PageControl/examples/PageControlTypicalUseExample.swift#L55)

Reply:
K!